Switching to draft as it confused `pipewire`:

```
pipewire> ../spa/plugins/bluez5/a2dp-codec-ldac.c:16:10: fatal error: ldacBT.h: No such file or directory
pipewire>    16 | #include <ldacBT.h>
pipewire>       |          ^~~~~~~~~~
```

--- a/pkgs/development/libraries/ldacbt/default.nix
+++ b/pkgs/development/libraries/ldacbt/default.nix
@@ -15,8 +15,15 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
+  outputs = [ "out" "dev" ];
+
   nativeBuildInputs = [
     cmake
+  ];
+
+  cmakeFlags = [
+    # CMakeLists.txt by default points to $out
+    "-DINSTALL_INCLUDEDIR=${placeholder "dev"}/include"
   ];
 
   meta = with lib; {

